src/assignment/views/AssignmentEdit.tsx

```diff
@@ -99,21 +93,21 @@ interface AssignmentEditProps extends RouteComponentProps {
 	loginState: LoginResponse | null;
 }
 
-let currentAssignment: Partial<Assignment>;
-let setCurrentAssignment: (newAssignment: Partial<Assignment>) => void;
-let initialAssignment: Partial<Assignment>;
-let setInitialAssignment: (newAssignment: Partial<Assignment>) => void;
+let currentAssignment: Partial<Avo.Assignment.Assignment>;
```

Comment:
Kan iemand mij deze nog eens uiteggen waarom hier `let`'s voor gebruikt worden 😅?

Reply:
Dat was een workaround om dat issue op te lossen met de state die soms een oudere state gebruikte
maar mogelijks is dat probleem verholpen door slechts 1 useEffect call meer te gebruiken

Ik zou hier een TODO aan toevoegen en dan test ik het eerst eens grondig uit als de database terug werkt.